### DISCUSSION TITLE
Support Wezterm?

### DISCUSSION THREAD

**Discussion by zenithyr** [jpalardy/vim-slime#337] [Ideas]
[Wezterm](https://wezfurlong.org/wezterm/) is a super awesome terminal I recently found on OSX. Actually I moved away from alacritty + tmux because wezterm has the pane/window by itself.

Unfortunately my next favorite vim-slime doesn't have wezterm as a valid target. I wonder if it's possible to extend the support there?

**Comment by jpalardy** [comment]
Hi @zenithyr 

There's #333 . Maybe we can re-open that and clean it a bit?

**Reply by zenithyr** [reply]
Oops my apologies. Should have found that thread earlier. Will update there.
